packages/runtime/datastore-hot-swap/src/hotSwapDataStoreRuntime.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { FluidObject } from "@fluidframework/core-interfaces";
+import {
+	IChannel,
+	IChannelFactory,
+	IFluidDataStoreRuntime,
+} from "@fluidframework/datastore-definitions";
+import { assert } from "@fluidframework/core-utils";
+import { FluidDataStoreRuntime, ISharedObjectRegistry } from "@fluidframework/datastore";
+import { ILocalChannelContext, IModifiableFluidDataStoreContext } from "./types";
+
+/**
+ * DO NOT USE THIS CLASS DIRECTLY.
+ */
+export class UnsafeHotSwapFluidDataStoreRuntime extends FluidDataStoreRuntime {
```

Comment:
Don't extend, just copy over the whole package and modify.  Since the end result is going to be porting the modifications back to the original DataStoreRuntime, this will mean the experimental package will more closely resemble the final state than if you extend it.

Reply:
I'm pretty sure we're not going to be porting these changes back to the original `FluidDataStoreRuntime`